controllers/app.controller.js

```diff
@@ -0,0 +1,10 @@
+const {fetchCategories} = require("../models/app.model")
+
+const getCategory = (request, response, next) => {
+    fetchCategories().then((categoryArr) => {
+        response.status(200).send({Categories : categoryArr})
```

Comment:
Categories shouldn't have a capital letter and you could call it categories as the parameter in the call back function in the .then  and then just sent it back  .send({categories}) so you send the array back on the key of the same name. If you don't understand that don't worry but please do change the capital letter.